src/tracks/Pattern.cpp

```diff
@@ -1037,16 +1037,13 @@ void PatternView::paintEvent( QPaintEvent * )
 			{
 				const int vol = n->getVolume();
 				p.drawPixmap( x, y, stepoffl );
+				p.drawPixmap( x, y, stepon0 );
 				if( vol > 0 )
```

Comment:
The only reason the statement might stay is CPU usage, but that is so minor that I don't think it should affect this.